src/features/common/hooks/events.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react';
```

Comment:
I think as this is a Typescript file that uses hooks, the extension needs to  be `.tsx`